src/overlaybd/zfile/compressor.cpp

```diff
@@ -179,8 +186,6 @@ class LZ4Compressor : public BaseCompressor {
         max_dst_size = LZ4_compressBound(src_blk_size);
 #ifdef ENABLE_QAT
         if (check_qat()) {
-            pQat = new LZ4_qat_param();
-            qat_init(pQat);
             qat_enable = true;
```

Comment:
We may need to create a lot of decompressor instances, we should integrate check_qat() and session cache into a global qat object.